brouter-routing-app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,51 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <!-- some apps (bluemail) do not recognize the .brf file extention, startactivity+intent is done for attachement with text/plain -->
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="text/plain"/>
+            </intent-filter>
+
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+                <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER" />
+            </intent-filter>
+            <!-- general intent to register .brf files -->
+            <intent-filter
+                android:label="@string/app_name"
+                android:priority="50">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="file"/>
+                <data android:scheme="content"/>
+                <data android:host="*"/>
+                <data android:pathPattern=".*\\.brf" />
+                <data android:pathPattern=".*\\..*\\.brf" />
+                <data android:pathPattern=".*\\..*\\..*\\.brf" />
+                <data android:pathPattern=".*\\..*\\..*\\..*\\.brf" />
+                <data android:pathPattern=".*\\..*\\..*\\..*\\..*\\.brf" />
```

Comment:
Are those multiple pathPattern necessary?

Reply:
I think so, as it is used by quite every registration ...

Reply:
Hello,
most of the apps are using "view" in the intent by clicking the attachement (we have no influence on this!).
So, to accept the intent in Brouter we have to use VIEW.

I did not test an import of "multiple profiles", possibly an error will occurs.

The log statements was for debug / test, they can be migrated (or cleared by need)

Reply:
Not sure, but all the apps I could look at do so ....